Kitodo/src/main/resources/xslt/kitodo2mods.xsl

```diff
@@ -0,0 +1,150 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<!--
+ *
+ * (c) Kitodo. Key to digital objects e. V. <contact@kitodo.org>
+ *
+ * This file is part of the Kitodo project.
+ *
+ * It is licensed under GNU General Public License version 3 or later.
+ *
+ * For the full copyright and license information, please read the
+ * GPL3-License.txt file that was distributed with this source code.
+ *
+-->
+
+<xsl:stylesheet version="2.0"
+                xmlns:xsl="http://www.w3.org/1999/XSL/Transform"
+                xmlns:mets="http://www.loc.gov/METS/"
+                xmlns:mods="http://www.loc.gov/mods/v3"
+                xmlns:kitodo="http://meta.kitodo.org/v1/"
+>
+    <xsl:output method="xml" indent="yes" encoding="utf-8"/>
+    <xsl:strip-space elements="*"/>
+
+    <!-- ### TitleDocMain ### -->
+    <xsl:template match="kitodo:metadata[@name='TitleDocMain']">
+        <mods:mods>
+            <mods:titleInfo>
+                <mods:title>
+                    <xsl:value-of select="normalize-space()" />
+                </mods:title>
+            </mods:titleInfo>
+        </mods:mods>
+    </xsl:template>
+
+    <!-- ### Author, slub_Recipient ### -->
+    <xsl:template match="kitodo:metadataGroup[@name='Person']">
+        <xsl:variable name="role" select="../kitodo:metadata[@name='Role']" />
+        <xsl:variable name="last_name" select="../kitodo:metadata[@name='lastName']" />
+        <xsl:variable name="first_name" select="../kitodo:metadata[@name='firstName']" />
+        <xsl:choose>
+            <xsl:when test="$role='author'">
+                <mods:mods>
+                    <mods:name>
+                        <mods:namePart type="family">
+                            <xsl:value-of select="$last_name" />
+                        </mods:namePart>
+                        <mods:namePart type="given">
+                            <xsl:value-of select="$first_name" />
+                        </mods:namePart>
+                        <mods:displayForm>
+                            <xsl:value-of select="$last_name" />, <xsl:value-of select="$first_name" />
+                        </mods:displayForm>
+                        <mods:role>
+                            <mods:roleTerm type="code" authority="marcrelator">
+                                aut
+                            </mods:roleTerm>
+                            <mods:roleTerm type="text" authority="marcrelator">
+                                Author
+                            </mods:roleTerm>
+                        </mods:role>
+                    </mods:name>
+                </mods:mods>
+            </xsl:when>
+        </xsl:choose>
+    </xsl:template>
+
+    <!-- ### PlaceOfPublication ### -->
+    <xsl:template match="kitodo:metadata[@name='PlaceOfPublication']">
+        <mods:mods>
+            <mods:originInfo>
```

Comment:
`<mods:originInfo>` should have a `@eventType="publication"` to distinguish it from e.g. reproduction metadata.
```suggestion
            <mods:originInfo eventType="publication">
```